source/administration/important-upgrade-notes.rst

```diff
@@ -8,7 +8,10 @@ Important Upgrade Notes
    Support for Mattermost Server v5.19 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ has come to the end of its lifecycle. Upgrading to v5.25 or later is required.
 
 .. important::
-   TLS versions 1.0 and 1.1 have been deprecated by browser vendors. In an upcoming release (version TBD), mmctl will return an error when connected to Mattermost servers deployed with these TLS versions and System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
+   TLS versions 1.0 and 1.1 have been deprecated by browser vendors. In Mattermost Server v5.32 (February 16), mmctl will return an error when connected to Mattermost servers deployed with these TLS versions and System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
+
+.. important::
+   Customers using the ``mattermost`` binary with the “--platform” flag, or using the ``platform`` binary directly, to run the Mattermost server application via a systemd file or custom script will be required to update to use only the ``mattermost`` binary as the “--platform” flag and the ``platform`` binary will be deprecated in a future release.
```

Comment:
Recommended edits: 

The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the ``mattermost`` binary.